### PR TITLE
Move philosophy to bottom

### PR DESCRIPTION
**Pull request by EVogel1999**
Moves the clark philosophy section on the homepage to the bottom of the page so that recently released learning objects appear as the first option on the homepage.

### CODE REVIEW

**Comment by EVogel1999** [comment]
image
image

**Comment by Olaniyan99** [comment]
can we also get a mobile view please.

File: src/app/cube/home/home.component.scss [hunk 1]
```diff
@@ -18,7 +18,7 @@
     width: 100%;
     height: 100vw;
     z-index: 1;
-    background: white;
+    background: #f6f9fc;
```
Comment (pzalep1) [inline src/app/cube/home/home.component.scss]: Do we have a variable for this?

**Comment by Olaniyan99** [comment]
Screen Shot 2020-02-28 at 9 59 56 AM (2)

Also i noticed the "Quality curriculum" module is a little off.

File: src/app/cube/home/home.component.scss (hunk 1 above)
Comment (EVogel1999) [reply src/app/cube/home/home.component.scss]: No, this is remanent of before we had a global sass variable for almost every color on our site.  I can add it if you want.

**Comment by EVogel1999** [comment]
> Screen Shot 2020-02-28 at 9 59 56 AM (2)
> 
> Also i noticed the "Quality curriculum" module is a little off.

Good catch!  I didn't notice that.  I think that is how it displays if the screen is too zoomed out.  That picture was taken at 25% zoom out I believe.  At 50% on chrome I see this:
image

File: src/app/cube/home/home.component.scss (hunk 1 above)
Comment (pzalep1) [reply src/app/cube/home/home.component.scss]: Let's use the variables. This makes it easier if we ever need adjust colors for some reason. This way we only have to update the white color we use in one place versus the 100+ stylesheets we have in the client.

**Pull request merged**